projects/swimlane/ngx-charts/package.json

```diff
@@ -45,6 +45,7 @@
     "rxjs": "^6.5.3 || ^7.4.0"
   },
   "dependencies": {
+    "@types/d3-shape": "^2.0.0",
```

Comment:
types should be always a devDependencies

Reply:
Thanks for catching that. I will update it to be a dev dep